packaging/linux/buildpackage.sh

```diff
@@ -110,63 +104,70 @@ else
+rm -rf "${PACKAGING_APPIMAGE_DEPENDENCIES_SOURCE}"
 
 # Add launcher and icons
-sed "s/{MODID}/${MOD_ID}/g" include/AppRun.in | sed "s/{MODNAME}/${PACKAGING_DISPLAY_NAME}/g" > AppRun.temp
-install -m 0755 AppRun.temp "${BUILTDIR}/AppRun"
+sed "s/{MODID}/${MOD_ID}/g" "${TEMPLATE_ROOT}/${ENGINE_DIRECTORY}/packaging/linux/AppRun.in" | sed "s/{MODNAME}/${PACKAGING_DISPLAY_NAME}/g" > "${APPDIR}/AppRun"
+chmod 0755 "${APPDIR}/AppRun"
+
+if [ -n "${PACKAGING_DISCORD_APPID}" ]; then
+	sed "s/{DISCORDAPPID}/${PACKAGING_DISCORD_APPID}/g" "${TEMPLATE_ROOT}/${ENGINE_DIRECTORY}/packaging/linux/openra.desktop.discord.in" > temp.desktop.in
+	sed "s/{DISCORDAPPID}/${PACKAGING_DISCORD_APPID}/g" "${TEMPLATE_ROOT}/${ENGINE_DIRECTORY}/packaging/linux/openra-mimeinfo.xml.discord.in" > temp.xml.in
+else
+	cp "${TEMPLATE_ROOT}/${ENGINE_DIRECTORY}/packaging/linux/openra.desktop.in" temp.desktop.in
+	cp "${TEMPLATE_ROOT}/${ENGINE_DIRECTORY}/packaging/linux/openra-mimeinfo.xml.in" temp.xml.in
+fi
 
-sed "s/{MODID}/${MOD_ID}/g" include/mod.desktop.in | sed "s/{MODNAME}/${PACKAGING_DISPLAY_NAME}/g" | sed "s/{TAG}/${TAG}/g" > temp.desktop
-install -Dm 0755 temp.desktop "${BUILTDIR}/usr/share/applications/openra-${MOD_ID}.desktop"
-install -m 0755 temp.desktop "${BUILTDIR}/openra-${MOD_ID}.desktop"
+mkdir -p "${APPDIR}/usr/share/applications"
+chmod 0755 temp.desktop.in
+sed "s/{MODID}/${MOD_ID}/g" temp.desktop.in | sed "s/{MODNAME}/${PACKAGING_DISPLAY_NAME}/g" | sed "s/{TAG}/${TAG}/g" > "${APPDIR}/usr/share/applications/openra-${MOD_ID}.desktop"
+cp "${APPDIR}/usr/share/applications/openra-${MOD_ID}.desktop" "${APPDIR}/openra-${MOD_ID}.desktop"
+rm temp.desktop.in
 
-sed "s/{MODID}/${MOD_ID}/g" include/mod-mimeinfo.xml.in | sed "s/{TAG}/${TAG}/g" > temp.xml
-install -Dm 0755 temp.xml "${BUILTDIR}/usr/share/mime/packages/openra-${MOD_ID}.xml"
+mkdir -p "${APPDIR}/usr/share/mime/packages"
+chmod 0755 temp.xml.in
+sed "s/{MODID}/${MOD_ID}/g" temp.xml.in | sed "s/{TAG}/${TAG}/g" > "${APPDIR}/usr/share/mime/packages/openra-${MOD_ID}.xml"
+rm temp.xml.in
 
-if [ -f "${PACKAGING_DIR}/mod_scalable.svg" ]; then
-  install -Dm644 "${PACKAGING_DIR}/mod_scalable.svg" "${BUILTDIR}/usr/share/icons/hicolor/scalable/apps/openra-${MOD_ID}.svg"
+if [ -f "${ARTWORK_DIR}/icon_scalable.svg" ]; then
+	install -Dm644 "${ARTWORK_DIR}/icon_scalable.svg" "${APPDIR}/usr/share/icons/hicolor/scalable/apps/openra-${MOD_ID}.svg"
 fi
 
 for i in 16x16 32x32 48x48 64x64 128x128 256x256 512x512 1024x1024; do
-  if [ -f "${ARTWORK_DIR}/icon_${i}.png" ]; then
-    install -Dm644 "${ARTWORK_DIR}/icon_${i}.png" "${BUILTDIR}/usr/share/icons/hicolor/${i}/apps/openra-${MOD_ID}.png"
-    install -m644 "${ARTWORK_DIR}/icon_${i}.png" "${BUILTDIR}/openra-${MOD_ID}.png"
-  fi
+	if [ -f "${ARTWORK_DIR}/icon_${i}.png" ]; then
+		install -Dm644 "${ARTWORK_DIR}/icon_${i}.png" "${APPDIR}/usr/share/icons/hicolor/${i}/apps/openra-${MOD_ID}.png"
+		install -m644 "${ARTWORK_DIR}/icon_${i}.png" "${APPDIR}/openra-${MOD_ID}.png"
```

Comment:
This command will overwrite continously (on purpose?) and it is the wrong folder for icons. Do you want to install into the now obsolete `/usr/share/pixmaps` instead?

Reply:
This is for https://docs.appimage.org/reference/appdir.html#root-icon.

Ovewriting the icon each time is the simplest way to ensure that we always copy the highest resolution version that is available.